src/status_im/contexts/profile/contact/header/view.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im.contexts.profile.contact.header.view
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [quo.theme :as quo.theme]
```

Comment:
nit `[quo.theme]`

Reply:
wdym?

Reply:
You don't need the :as because we're not renaming it